packages/fxa-content-server/app/scripts/templates/connect_another_device.mustache

```diff
@@ -2,7 +2,10 @@
   <header class="hidden" id="fxa-connect-another-device-header"><h1>{{#t}}Connect another device{{/t}}</h1></header>
   {{#showSuccessMessage}}
     {{#isSignedIn}}
-      <div class="success success-authenticated visible">{{#t}}This Firefox is connected{{/t}}</div>
+      <div id="fxa-connected" class="flex flex-nowrap justify-center">
+        <div id="circle-check" class="bg-[#B3FFE3] rounded-[50%] shrink-0 h-8 w-8 mt-3.5 relative"></div>
```

Comment:
@chenba Haven't had a chance to check into your branch to actually review, but can we use Tailwind color values, as well as Tailwind values for padding or anything else? The designs should be consistent with FxA's design guide as far as colors and spacing goes. However, I recognize the circle here might be a special green color, I figured the entire thing would be an SVG and images don't adhere to those guidelines, and from the code it seems like it's just the checkmark part that was provided so that ones okay IMO but not sure about the text colors done below this. We can ask jikim (username on Slack) any clarification questions like if something is showing as `18px`, that should be `16px` or `20px`.

I wouldn't block r+ on TW styles though FWIW since I can tweak 'em in a ticket I'm about to start anyways and we'll have a second look at them later whenever this is refactored to React. Thanks for refactoring these bits. 😁

Btw I was going to suggest adding `role="img"` as well as `aria-label=` BUT I felt torn about it and asked Meghan and we had a quick back and forth and think it's [decorative](https://www.w3.org/WAI/tutorials/images/decorative/) so this seems fine.

Reply:
~~I can ask for an SVG of the green check.  Couldn't hurt.~~  zomg I found the export feature in Figma.  Can't believe I spent the time using CSS...

#737373 is a Photon grey that's used in content-server.  Thought I was being consistent.  But yes, I can replace it.